cmd/noms/noms_sync_test.go

```diff
@@ -63,6 +65,7 @@ func (s *nomsSyncTestSuite) TestSync() {
 	sout, _ := s.MustRun(main, []string{"sync", sourceSpec, sinkDatasetSpec})
 
 	s.Regexp("Created", sout)
+	os.Mkdir(db2dir, 0777)
```

Comment:
Instead of replicating this everywhere, how about modifying `ClientTestSuite::SetupSuite()` to create the directory? So that it will already be created when every test tries to use it.